src/sentry/static/sentry/app/views/performance/vitalDetail/utils.tsx

```diff
@@ -0,0 +1,116 @@
+import {Query, Location} from 'history';
+
+import {getAggregateAlias, WebVital} from 'app/utils/discover/fields';
+import {decodeScalar} from 'app/utils/queryString';
+import {Series} from 'app/types/echarts';
+
+import {WEB_VITAL_DETAILS} from '../transactionVitals/constants';
+
+export function generateVitalDetailRoute({orgSlug}: {orgSlug: string}): string {
+  return `/organizations/${orgSlug}/performance/vitaldetail/`;
+}
+
+export const vitalsThresholdFields = {
+  [WebVital.FP]: 'count_at_least(measurements.fp, 3000)',
+  [WebVital.FCP]: 'count_at_least(measurements.fcp, 3000)',
+  [WebVital.LCP]: 'count_at_least(measurements.lcp, 4000)',
+  [WebVital.FID]: 'count_at_least(measurements.fid, 300)',
+  [WebVital.CLS]: 'count_at_least(measurements.cls, 0.25)',
+};
```

Comment:
Would be good if these thresholds are imported from this [file](https://github.com/getsentry/sentry/blob/b09bd2d62edf37a9723eff277fc49bcc8c41e3a0/src/sentry/static/sentry/app/views/performance/transactionVitals/constants.tsx#L12) instead of hard coded here.

Reply:
I thought about adding it there, but these vitals will need to be extended in the interim to backend 'vitals' as well, I didn't want to conflate them.